converter/manifoldSyntax.ts

```diff
@@ -11,14 +11,30 @@ const openscadModulesManifold: commonSyntax.OpenScadModules = {
       return `cube(${params.size},${params.center || false})\n`
     }
   },
+  cylinder: {
+    openscadParams: ['h', 'r1', 'r2', 'center', '$fn'],
+    code: (params) => {
+      const h = params.h
+      const r1 = params.r1 || params.r || (params.d && `${params.d} / 2`) || (params.d1 && `${params.d1} / 2`) || '1'
+      const r2 = params.r2 || (params.d2 && `${params.d2} / 2`)
+      const center = params.center || 'false'
+
+      if (r2 || params.$fn || params.center) {
+        return `cylinder(${h}, ${r1}, ${r2 || '0'}, ${params.$fn || 'null'}, ${center})\n`
+      } else {
+        return `cylinder(${h}, ${r1})\n`
```

Comment:
@pca006132 - How does this approach compare to what you were imagining for an OpenSCAD -> ManifoldCAD transcriber? Do you think building out this syntax will get us to automatic import of some common OpenSCAD libraries?

Reply:
This looks fine to me.